graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java

```diff
@@ -62,40 +63,77 @@ public static void main(String[] args) throws Exception {
     });
   }
 
-  private static String PYTHON = "python";
-  // Get the location of the graalpy executable
-  private static String VENV_EXECUTABLE = Main.class.getClassLoader()
-      .getResource(Paths.get("vfs", "venv", "bin", "graalpy").toString()).getPath();
-  // Name of the file containing Python source
-  private static String SOURCE_FILE_NAME = "PyfigletWrapper.py";
+  private static final String PYTHON = "python";
+  private static final String VENV_PREFIX = "/vfs/venv";
+  private static final String HOME_PREFIX = "/vfs/home";
+  private static final String PROJ_PREFIX = "/vfs/proj";
+
+  public static Context getContext() {
+    VirtualFileSystem vfs = VirtualFileSystem.newBuilder()
+      .extractFilter(p -> {
+        String s = p.toString();
```

Comment:
Are you sure this is needed? I think there is `Path#endsWith(String other)`.

Reply:
This is just the generated code from our archetype, I would leave it as is?